investment_tracking/urls.py

```diff
@@ -1,7 +1,8 @@
 from django.urls import path
-from .views import InvestmentTrackingSaveView
+from .views import *
 
```

Comment:
it is better not to use "*" for import, but to list the required libraries

Reply:
Done